### PR TITLE
Use own octavia user for other services too

### PR DESCRIPTION
**Pull request by weinimo**
Change according to proposals:

 https://github.com/openstack-k8s-operators/cinder-operator/pull/107
 https://github.com/openstack-k8s-operators/neutron-operator/pull/93

### CODE REVIEW

File: templates/octaviaapi/config/octavia.conf [hunk 1]
```diff
@@ -89,7 +89,7 @@ auth_type = password
 project_domain_name = Default
 user_domain_name = default
 region_name = regionOne
-username = nova
+username = {{ .ServiceUser }}
```
Comment (kajinamit) [inline templates/octaviaapi/config/octavia.conf:92]: Do we really need this user credential ? I guess octavia just uses the one in service_auth instead.

File: templates/octaviaapi/config/octavia.conf [hunk 2]
```diff
@@ -102,7 +102,7 @@ auth_type = password
 project_domain_name = Default
 user_domain_name = default
 region_name = regionOne
-username = neutron
+username = {{ .ServiceUser }}
```
Comment (kajinamit) [inline templates/octaviaapi/config/octavia.conf:105]: ditto. I don't think we need the credential in [neutron] section at least as of zed.

**Review by kajinamit — CHANGES_REQUESTED** [review]
You probably want to update ./templates/octaviaapi/bin/init.sh to set the password but I'm skeptical about the credentials parameters in nova and neutron section. Do you care to double-check that ?

File: templates/octaviaapi/config/octavia.conf (hunk 1 above)
Comment (gthiemonge) [reply templates/octaviaapi/config/octavia.conf:92]: I agree, service_auth should be enough

**Comment by weinimo** [comment]
Superseded by #60. Thanks

**Pull request closed without merging**